### ISSUE TITLE
HTML Editor for Email templates

### ISSUE THREAD

**Issue by bethlakshmi** [bethlakshmi/GBE2#1124]
Set up a way to edit email templates with a WYSIWYG HTML editor so that the user doesn't have to know HTML to code.  This would happen on the regular pages (not Admin).

Flow would be something like:
- if user has a viable privilege,
- there's a menu item for Manage -> Auto-Gen Email 
- when that option is selected, the user gets a list of Email Templates they have access to.
- user selects applicable template and gets:
   -  a regular text box for regular text
   - HTML WYSIWYG editor
   - preview of HTML (not editable)
   - guide of what special tags are available for use in the email (not editable)
- user can Save or Cancel
- Save updates the template for all future email.

Suggestion for who can do what:
- bid coordinators can do all emails relating to their bid

**Comment by bethlakshmi** [comment]
Question to @burlexpo --

How do users navigate to this...?  They need to pick what template they want to edit, and there's no point to them making up a new template name (right now) because all the auto-email code expects a *specific* name - so new templates don't get used by anything.

Idea 1:
- On the Special menu, in "Manage"
- Add a new item called "Auto-Generated Email" and give it a sublist of the template names.  
- use logic to make sure that only the bid coordinator gets the option to edit those bid types.  

I'd also make ALL template types available to admins.

Idea 2:
- There's a switch for Special -> Manage -> Auto-Generated Email (same as above)
- BUT instead of a submenu, clicking this takes you to a page.  
- the page lets you pick the template, but it also gives you a description of where and when this email type is used.

I'm also up for other navigation thoughts.  Idea 1 is fewer clicks, Idea 2 is more help for the user.

**Comment by burlexpo** [comment]
I like option #2

**Comment by burlexpo** [comment]
@bethlakshmi I created an act (Test the Template), submitted it, and accepted for The Bordello.  It's been 15 minutes and I have received an acceptance email.  Bug?

**Comment by bethlakshmi** [comment]
This is as designed.  It's a very bad idea to send email from a TEST system to actual participants, so I rigged a lookup in the email code to check for the DEBUG setting.  If we're in debug mode, email is sent to the admins and not actual participants.

Info@burlesque-expo.com is one of the Admins.

I did this after I accidently sent legimitate looking email to actual performers when I was trying to test bug fixes on gbeadmin.  I always wrote them to apologize, but it was embarassing so I decided to eliminate the potential.
